skglm/utils.py

```diff
@@ -43,6 +43,57 @@ def box_proj(x, low, up):
         return x
 
 
+@njit
+def value_MCP(w, alpha, gamma):
```

Comment:
Is there a reason behind defining value and prox functions in a separate file (``utils.py``) and not within the class definition?

Reply:
yes, they are used both in single task and group penalties